modules/oadp-deleting-backups-using-velero.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups-doc.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-deleting-backups-using-velero_{context}"]
+= Deleting a backup by using Velero command
```

Comment:
Same here re: adding an intro sentence to the procedure. But also in the title, should it be "by using the velero command" (lowercase for the command name, but no backticks because it's a heading) or "by using the Velero CLI" more generally? I might opt for the latter, FWIW.

```suggestion
= Deleting a backup by using the Velero CLI

You can delete a backup by using the Velero CLI.
```